server/channels/store/storetest/user_store.go

```diff
@@ -4471,23 +4472,56 @@ func testUserStoreAnalyticsGetInactiveUsersCount(t *testing.T, rctx request.CTX,
 	u1.Email = MakeEmail()
 	_, err := ss.User().Save(rctx, u1)
 	require.NoError(t, err)
-	defer func() { require.NoError(t, ss.User().PermanentDelete(rctx, u1.Id)) }()
+	t.Cleanup(func() { require.NoError(t, ss.User().PermanentDelete(rctx, u1.Id)) })
 
 	count, err := ss.User().AnalyticsGetInactiveUsersCount()
 	require.NoError(t, err)
-
+	require.Equal(t, count, 0, "No users should have been inactive yet")
```

Comment:
`require.Equal` is type sensitive :confused: 
```suggestion
	require.Equal(t, count, int64(0), "No users should have been inactive yet")

```